integration/test/app/basic/simple_test.go

```diff
@@ -65,6 +65,17 @@ func TestAppLifecycle(t *testing.T) {
 		Namespace: namespace,
 	}
 
+	{
```

Comment:
e2e would install tiller unless it's using `inCluster` mode, hence we install this manually in case of basic test.